<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="l">
    <item>
     <widget class="QLabel" name="lblKodeBarang">
      <property name="text">
       <string>Kode Barang</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtKodeBarang"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hlytNamaBarang">
    <item>
     <widget class="QLabel" name="lblNamaBarang">
      <property name="text">
       <string>Nama Barang</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtNamaBarang">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="l_2">
    <item>
     <widget class="QLabel" name="lblHarga">
      <property name="text">
       <string>Harga           </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtHarga"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hlytNamaBarang_2">
    <item>
     <widget class="QLabel" name="lblStok">
      <property name="text">
       <string>Stok Barang</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtStok">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tblwgBarang">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>621</width>
     <height>361</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <column>
    <property name="text">
     <string>Kode Barang</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nama Barang</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Harga Barang</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Stok</string>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>499</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="btnSimpan">
      <property name="text">
       <string>Baru</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnEdit">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDelete">
      <property name="text">
       <string>Hapus</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>500</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hlytNamaBarang_3">
    <item>
     <widget class="QLabel" name="lblNamaBarangCari">
      <property name="text">
       <string>Nama Barang</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="txtCari">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCari">
      <property name="text">
       <string>Cari</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
